docs/ml/_index.md

```diff
@@ -16,19 +16,55 @@ menuindent: true
 {{<imgproc src="/ml/training.png" class="alignright" resize="400x" declaredimensions=true alt="ML training">}}
 
 Machine Learning (ML) provides your machines with the ability to adjust its behavior based on models that recognize patterns or make predictions.
+
 Common use cases include:
 
-- Object detection and classification which enable machines to detect people, animals, plants, or other objects with bounding boxes, and to perform actions when they are detected.
+- Object detection, which enables machines to detect people, animals, plants, or other objects with bounding boxes, and to perform actions when they are detected.
+- Object classification, which enables machines to separate people, animals, plants, or other objects into predefined categories based on their characteristics, and to perform different actions based on the classes of objects.
 - Speech recognition, natural language processing, and speech synthesis, which enable machines to verbally communicate with us.
 
-However, your machine can make use of machine learning with nearly any kind of data.
+For other use cases, consider [creating custom functionality with a module](/registry/create/).
+
+Viam provides two services that enable machine learning capabilities: the [ML model](/ml/deploy/) service and the [Computer Vision](/ml/vision/) service.
+
+The ML model service deploys and runs a machine learning model, such as a TensorFlow or ONNX model, on your machine and makes its output accessible to other services.
+One service that is built on top of the ML model service is the [Computer Vision](/ml/vision/mlmodel/) `mlmodel` service which can detect or classify objects.
+As a detector, the service interprets image data from images on your computer or a [camera](/components/camera/) and draws bounding boxes around objects.
+As a classifier, the service returns class labels and confidence score based off the [inferences](/ml/deploy/#infer) the underlying ML model makes from image data.
 
-Viam supports:
+See [Use machine learning with your machine](#use-machine-learning-with-your-machine) for step by step instructions.
+
+## Example tutorials
```

Comment:
@npentrel Incorporated this-- don't mind it, essentially just moving the tutorials sections at the bottom up (removed that as duplicating). Visually it looks better. I added a fourth tutorial to make it a square if that's ok

Reply:
Sounds good. Would be nice to add the preview image to the manual card - sorry I forgot that in my suggestion. I'll approve already though